pkg/syncer/util.go

```diff
@@ -95,3 +95,14 @@ func getQueryResults(ctx context.Context, volumeIds []cnstypes.CnsVolumeId, clus
 	}
 	return allQueryResults, nil
 }
+
+// GetVolumeID returns volume handle if successfully found by the volumeMigrationService
+func GetVolumeID(ctx context.Context, pv *v1.PersistentVolume) (string, error) {
+	log := logger.GetLogger(ctx)
+	volumeHandle, err := volumeMigrationService.GetVolumeID(ctx, pv.Spec.VsphereVolume.VolumePath)
```

Comment:
Can we handle for nil pointer check before accessing `pv.Spec.VsphereVolume.VolumePath`. Since this is util, we should handle nil check here, otherwise, all callers need to make validate for NPE. You can also supply volumepath string in this util function.

Reply:
Done